### PR TITLE
TCVP-1511 Upgrade spring boot starter version

### PR DESCRIPTION
**Pull request by karahanb**
# Description

This PR includes the following proposed change(s):

- [TCVP-1511](https://justice.gov.bc.ca/jira/browse/TCVP-1511)
- Changed all entity data models having primary key as UUID type properties to Long type PK ID and updated related methods and JUnit tests accordingly since UUID serializes differently in the new H2 2.x database with the new spring 2.7.0 and breaks unit tests and some db operations.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/oracle-data-api/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.6.4</version>
+		<version>2.7.0</version>
```
Comment (KDNeufeld) [inline src/backend/oracle-data-api/pom.xml:9]: Can we actually pull out the changes to the pom.xml from this PR?  This way we can separate the UUID change from the spring upgrade PR. Once this PR has been merged into develop, we can rebase the dependabot PR and merge it in.

**Review by KDNeufeld — CHANGES_REQUESTED** [review]
Looks great.  I would just separate the model changes from the spring upgrade (pom.xml changes)

File: src/backend/oracle-data-api/pom.xml (hunk 1 above)
Comment (karahanb) [reply src/backend/oracle-data-api/pom.xml:9]: Sure, will change this back to 2.6.4 and get the spring upgrade from the dependabot PR after merging this one.

File: src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Dispute.java [hunk 2]
```diff
@@ -46,213 +42,221 @@
...
+	/**
+	 * The provincial court hearing location named on the violation ticket.
+	 */
+	@Column
+	@Schema(nullable = true)
+	private String provincialCourtHearingLocation;
+
+	/**
+	 * The date and time the violation ticket was issue. Time must only be hours and
+	 * minutes.
```
[the remark this answers is not in the document]
Comment (karahanb) [reply src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Dispute.java:67]: Changed that to 150 characters.

**Review by KDNeufeld — APPROVED** [review]
Awesome work.

**Pull request merged**